setup.py

```diff
@@ -58,7 +58,7 @@ def desc():
         "Flask-SQLAlchemy>=2.4, <3",
         "Flask-WTF>=0.14.2, <0.15.0",
         "Flask-JWT-Extended>=3.18, <4",
-        "jsonschema>=3.0.1, <4",
+        "jsonschema==4.0.0",  # pinning version for now
```

Comment:
let's not pin this, since FAB is a base package, do something like ">=3.0.1, <5", so we don't automaticaly included not tested major breaking changes